R/tilt.company.match-package.R

```diff
@@ -0,0 +1,7 @@
+#' @keywords internal
```

Comment:
I am not familiar with his file, can you drop that notes for @lindadelacombaz and my education?